modules/kmm-firmware-support.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: CONCEPT
+[id="kmm-firmware-support_{context}"]
+= KMM firmware support
+
+Kernel modules sometimes need to load firmware files from the filesystem. KMM supports copying firmware files from the
+[Module Loader image] to the node's filesystem.
+
+The contents of `.spec.moduleLoader.container.modprobe.firmwarePath` are copied into `/var/lib/firmware` on the node before `modprobe` is called to insert the kernel module.
+
+All files and empty directories are removed from that location before `modprobe -r` is called to unload the kernel module, when the pod is terminated.
```

Comment:
Same suggestion with the `modprobe -r` part

Reply:
Based on @kelbrown20 's comment and your reply [here](https://github.com/openshift/openshift-docs/pull/58256#discussion_r1163266681), should the `modprobe` instance be then written as show?

> ... before running the `modprobe` command to insert the kernel module."
> ... before running the `modprobe -r` command to unload the kernel module, when the pod is terminated.